Modix.Bot/CommandContextExtensions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Threading.Tasks;
+using Discord;
+using Discord.Commands;
+
+namespace Modix
+{
+    public static class CommandContextExtensions
+    {
+        private static readonly Emoji checkmarkEmoji = new Emoji("✅");
+
+        public static Task AddConfirmation(this ICommandContext context) =>
```

Comment:
Super nitpicky, but it looks like the style for this repository is to put the `=>` beneath the method signature